src/Tutor.Infrastructure/Database/TutorContext.cs

```diff
@@ -116,6 +116,7 @@ private static void ConfigureKcMastery(ModelBuilder modelBuilder)
             modelBuilder.Entity<KnowledgeComponentMastery>()
                 .HasMany(kcm => kcm.AssessmentMasteries)
                 .WithOne();
+            modelBuilder.Entity<KnowledgeComponentMastery>().OwnsOne(kcm => kcm.SessionTracker).Ignore(tracker => tracker.Id);
```

Comment:
To map SessionTracker to the database. Since it has a one to one relationship with KCM, I wanted it to be mapped as part of the same table to avoid unnecessary joins.